scripts/propose_upgrade.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash 
+echo "Creating Proposal For Upgrade"
+echo "Enter Title:"
+read -r title
+echo "Enter Height:"
+read -r height
+echo "Enter From User Address:"
+read -r from
+echo "Submit Proposal To Upgrade";
+# Submit Proposal
+sonrd tx gov submit-proposal software-upgrade --upgrade-height $height --from $from --yes --title $title --description test upgrade
+echo "Enter the Proposal ID:"
+read -r proposal_id
+# Deposit For Proposal
+echo "Depositing Funds For Proposal ID: $proposal_id"
+sonrd tx gov deposit $proposal_id 10000000stake --from $from --yes
+# Vote for the Proposal
+echo "Voting Yes For Proposal ID: $proposal_id"
+sonrd tx gov vote $proposal_id yes --from $from --yes
```

Comment:
Yes, we need to Vote yes every time, in a production scenario the upgrade requires a majority of the network to vote Yes in order to pass the proposal. Since the upgrade is a gov proposal based.

Reply:
Well then the upgrade technically wouldn't pass, it needs a majority of the network to agree for it to be passed just like any other governance proposal.

Reply:
this script was meant to be used for the demo and if a validator wishes to approve a proposal they need to execute the last line from it and vote yes to the proposal for the upgrade for it to pass. If a majority of the validators agree to a proposal that proposal is automatically passed.

Reply:
Okay sgtm.